features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/KeyManagers/AddEditKeyManager.jsx

```diff
@@ -743,7 +743,7 @@ function AddEditKeyManager(props) {
                                             validating)
                                 || intl.formatMessage({
                                     id: 'KeyManagers.AddEditKeyManager.form.introspectionEndpoint.help',
-                                    defaultMessage: 'Ex: https://localhost:9443/oauth2/introspect',
+                                    defaultMessage: 'Ex https://localhost:9443/oauth2/introspect',
                                 })}
```

Comment:
Ex? are we changing all to Eg: ?

Reply:
I have changed it in the UI messages

Reply:
Shall we fix it in the code base as well? Because if someone does a production build again the values you changed in the en.json files will get overwritten.